config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
It still gets loaded... I am not saying this is a security hole, I am just saying it's an unnecessary risk. Also it pollutes the namespace. Now you have a `StyleguideController` class in the global target app namespace. You really want that?